nntrainer/tensor/cache_pool.h

```diff
@@ -14,9 +14,9 @@
 #ifndef __CACHE_POOL_H__
 #define __CACHE_POOL_H__
 
-#include <mutex>
-#include <memory_pool.h>
 #include <list>
+#include <memory_pool.h>
+#include <mutex>
```

Comment:
Could you seperate nntrainer's header file and stl header file?

Reply:
thanks for checking